js/index.js

```diff
@@ -0,0 +1,110 @@
+import data from './appData.js';
+
+const menuBtn = document.getElementById('menu-btn');
+const closeBtn = document.getElementById('close-btn');
+const menu = document.getElementById('nav-items');
```

Comment:
- [ optional] please note that on the desktop version, the mobile popup menu is not suppose to be visible. please consider fixing that
![Screenshot (468)](https://user-images.githubusercontent.com/65298048/192098416-9ec9464a-7fbf-4e48-8d45-e6191f75272a.png)